scratch-js/Sprite.mjs

```diff
@@ -365,11 +417,35 @@ export class Sprite extends SpriteBase {
           );
           return false;
       }
+    } else if (target instanceof Color) {
+      return this._project.renderer.checkColorCollision(this, target);
     }
 
     return this._project.renderer.checkSpriteCollision(this, target, fast);
   }
 
+  colorTouching(color, target) {
+    if (typeof target === "string") {
+      console.error(
+        `Cannot find target "${target}" in "touching". Did you mean to pass a sprite class instead?`
```

Comment:
We should probably give a similar error message if `color` is not an instance of `Color`